buildrunner/__init__.py

```diff
@@ -419,21 +419,29 @@ def get_cache_archive_file(self, cache_name, project_name=""):
         """
         Given a cache name determine the local file path.
         """
-        caches_root = self.global_config.get('caches-root', DEFAULT_CACHES_ROOT)
-        build_path = os.path.splitdrive(self.build_dir)[1]
-        if os.path.isabs(build_path):
-            build_path = build_path[1:]
+        def get_filename(caches_root, cache_name):
```

Comment:
This is only used one place, why have a sub-method here?

Reply:
It's used two places in the `try` and in the `except`. I just didn't want to duplicate the code.

Reply:
browser search, you have failed me for the last time...